### PR TITLE
Add delay round end to the server tab

### PR DESCRIPTION
**Pull request by dragomagol**
## About The Pull Request
Adds the Delay Round End button to the Server tab for admins.

## Why It's Good For The Game
It has always _only_ been in the Check Antags panel. This is very annoying and easy to forget/lose track of, especially when all related verbs are in the Server tab.

## Changelog

:cl:
admin: Added Delay Round End to the server tab
/:cl:

### CODE REVIEW

**Comment by dragomagol** [comment]
I'm about to fuck up your muscle memory by adding a new verb

**Comment by wesoda25** [comment]
Touching admin buttons, 7 years of bad luck

File: code/modules/admin/admin_verbs.dm [hunk 1]
```diff
@@ -250,6 +251,7 @@ GLOBAL_LIST_INIT(admin_verbs_hideable, list(
 	/datum/admins/proc/startnow,
 	/datum/admins/proc/restart,
 	/datum/admins/proc/delay,
+	/datum/admins/proc/delay_round_end,
```
Comment (LemonInTheDark) [inline code/modules/admin/admin_verbs.dm:254]: tell me, why does `admin_verbs_hideable` exist?

File: code/modules/admin/verbs/server.dm [hunk 2]
```diff
@@ -128,6 +128,30 @@
 		to_chat(usr, "<span class='warningplain'><font color='red'>Error: Start Now: Game has already started.</font></span>")
 	return FALSE
 
+/datum/admins/proc/delay_round_end()
+	set category="Server"
+	set desc="Prevent the server from restarting"
+	set name="Delay Round End"
```
Comment (LemonInTheDark) [inline code/modules/admin/verbs/server.dm:134]:
```suggestion
	set category = "Server"
	set desc = "Prevent the server from restarting"
	set name = "Delay Round End"
```

File: code/modules/admin/verbs/server.dm [hunk 3]
```diff
@@ -128,6 +128,30 @@
...
+	if(SSticker.delay_end)
```
Comment (LemonInTheDark) [inline code/modules/admin/verbs/server.dm:135]: This needs a permissions check, move that over too

**Review by LemonInTheDark — CHANGES_REQUESTED** [review]

File: code/modules/admin/admin_verbs.dm (hunk 1 above)
Comment (dragomagol) [reply code/modules/admin/admin_verbs.dm:254]: Genuinely no idea; would you prefer it be removed?

image

image

image

Comment (LemonInTheDark) [reply code/modules/admin/admin_verbs.dm:254]: yes

**Pull request merged**